packages/main/test/specs/Card.spec.js

```diff
@@ -44,4 +44,27 @@ describe("Card general interaction", () => {
 		assert.strictEqual(card2.getAttribute("aria-label"), EXPECTED_ARIA_LABEL2,
 			"The aria-label is correctly set internally.");
 	});
+
+	it("Tests internal aria-labelledby labeling", () => {
+		const card1 = browser.$("#card2").shadow$(".ui5-card-root");
+		const header = browser.$("#card2").shadow$(".ui5-card-header");
+		const card2 = browser.$("#card3").shadow$(".ui5-card-root");
+		const header2 = browser.$("#card3").shadow$(".ui5-card-header");
+		const EXPECTED_ARIA_LABELLEDBY_CARD = "ui5wc_20-heading ui5wc_20-desc";
```

Comment:
To make the test more robust you can use he built-in **"_id"** property - card1.getProperty("_id"), card2.getProperty("_id") to get these parts ui5wc_20 and ui5wc_21. 
Because, if someone adds more cards in the test page, these cards might get different IDs, f.e. ui5wc_23, ui5wc_24 as this number depends on  the component's definition order and every time the test might need to be adjusted as it would fail.

```js
const card1Id = card1.getProperty("_id");
const EXPECTED_ARIA_LABELLEDBY_CARD = `${card1Id}-heading ${card1Id}-desc`;
const EXPECTED_ARIA_LABELLEDBY_HEADER = "${card1Id}-subheading ${card1Id}-status ${card1Id}-avatar";

const card2Id = card2.getProperty("_id");
...
```

Also, for reference how "_id" is used you can see NotificationList.spec.js -> "tests List Group Item ACC ariaLabelledBy".

Reply:
thanks for the feedback. :) applied to the change.